pkg/trace/pb/stats.proto

```diff
@@ -11,14 +9,14 @@ message ClientStatsPayload {
 	string hostname = 1;
 	string env = 2;
 	string version = 3;
-	repeated ClientStatsBucket stats = 4 [(gogoproto.nullable) = false];
+	repeated ClientStatsBucket stats = 4;
```

Comment:
why did you change this?
You can generate with gogoproto
// protoc -I=. -I=$GOPATH/src -I=$GOPATH/src/github.com/gogo/protobuf/protobuf --gogofaster_out=. ddsketch.proto stats.proto